MusicApp.Identity.Web/Middlewares/ErrorHandlingMiddleware.cs

```diff
@@ -0,0 +1,45 @@
+﻿using MusicApp.Identity.Domain.Exceptions;
+using System.Net;
+using System.Text.Json;
+
+namespace MusicApp.Identity.Web.Middlewares;
+
+public class ErrorHandlingMiddleware
+{
+    private readonly RequestDelegate _next;
+
+    public ErrorHandlingMiddleware(RequestDelegate next)
+    {
+        _next = next;
+    }
+
+    public async Task InvokeAsync(HttpContext context)
+    {
+        try
+        {
+            await _next(context);
+        }
+        catch (Exception ex)
```

Comment:
Is it only one case for exceptions?

Reply:
I can use UseExceptionHandler(). Do I need to use it?

Reply:
i mean, that you can have different types of exception and status codes will be different accroding to it